.github/workflows/run-tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y python3-setuptools nfs-common
-          sudo pip3 install --upgrade pip
+          sudo pip3 install --ignore-installed --upgrade pip
```

Comment:
out of curiosity, what kind of error does this fix?

Reply:
Right, I've seen that error as well. It's probably because `pip3` is installed through `apt`, and upgrading it through `pip3` won't work. I've opted to update the apt package instead.

Reply:
I guess this fix doesn't belong to this PR anyway.
BTW, there's another issue in the `main` killing the tests now.

Reply:
I think you're referring to the previous run when it failed: https://github.com/canonical/microk8s-community-addons/actions/runs/14036415249/job/39295392373

Seems like there's no issue anymore.